tensorflow_serving/servables/tensorflow/testdata/BUILD

```diff
@@ -93,6 +93,14 @@ filegroup(
     ),
 )
 
+filegroup(
+    name = "saved_model_half_plus_two_mkl",
+    srcs = glob(
+        ["saved_model_half_plus_two_mkl/**"],
+    ),
+)
+
+
```

Comment:
You'll need to remove this extra newline as well (that's the error we are seeing)

Reply:
yeah removed extra line, you guys may need to re-approve. @gautamvasudevan @netfs